assets/js/base/components/price-slider/index.js

```diff
@@ -81,16 +81,21 @@ const PriceSlider = ( {
 			};
 		}
 
+		// normalize to whatever is the closest step (because range input will
+		// only jump to the closest step in the range)
```

Comment:
Should we capitalize the comment and add a period at the end? It seems to be recommended in the coding standards:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#comments

Reply:
Good catch.  To bad this wasn't something picked up by linting :(